Runtime/RoadNetwork/Structure/RnRoad.cs

```diff
@@ -757,5 +763,56 @@ public static LineCrossPointResult GetLaneCrossPoints(this RnRoad self, LineSegm
                 .Concat(self.SideWalks.SelectMany(s => s.SideWays));
             return RnEx.GetLineIntersections(lineSegment, targetLines);
         }
+
+        /// <summary>
+        /// selfの内部をlineが取っているとしたときに, selfの両端のWayとlineの最も近い点との距離を返す
+        /// </summary>
+        /// <param name="self"></param>
+        /// <param name="line"></param>
+        /// <param name="distance"></param>
+        /// <returns></returns>
+        public static bool TryGetNearestDistance(this RnRoad self, RnLineString line, out float distance)
```

Comment:
最小の幅を計算しているかと思うのでこっちの方が適切？
```suggestion
        public static bool TryGetMinimumCrossingWidth(this RnRoad self, RnLineString line, out float distance)
```

Reply:
結果として最大幅となっていますが、あくまでlineとselfの両端の幅との最小の距離を返す関数なのでどちらかというと元の関数名の方が正しいです。
lineがselfの内部に無い時にも最も近い点との距離が返るので
ただ、それをわかりやすくするために関数名を変更しました。